pkg/handler/sign/hsm_pss/handler.go

```diff
@@ -0,0 +1,43 @@
+package hsm_pss
+
+import (
+	"crypto"
+	"net/http"
+
+	"github.com/miekg/pkcs11"
+	"github.com/open-component-model/signing-server/pkg/encoding"
+	"github.com/open-component-model/signing-server/pkg/handler/sign"
+	"github.com/open-component-model/signing-server/pkg/handler/sign/rsassa_pss"
+	signhttp "github.com/open-component-model/signing-server/pkg/http"
+)
+
+const (
+	// Algorithm defines the type for the RSA PSS signature algorithm.
+	Algorithm = rsassa_pss.Algorithm
```

Comment:
This constant isn't used really by anything other than the `Name()` function. Not sure it's needed. Is there an intention that this should be used outside of this package?

Reply:
if you want to rer to the algorithm name when calling the lib.

All handler use a constant at the beginning of the file to declare their name. This constant might be used outside, so why not making it public in all cases?